app/src/main/java/com/duckduckgo/app/onboarding/ui/page/extendedonboarding/ExtendedOnboardingFeatureToggles.kt

```diff
@@ -31,14 +37,61 @@ interface ExtendedOnboardingFeatureToggles {
     fun self(): Toggle
 
     @Toggle.DefaultValue(false)
-    @Experiment
     fun noBrowserCtas(): Toggle
 
     @Toggle.DefaultValue(false)
-    @Experiment
     fun privacyProCta(): Toggle
 
     @Toggle.DefaultValue(false)
     @Experiment
     fun highlights(): Toggle
+
+    @Toggle.DefaultValue(false)
+    fun testPrivacyProOnboardingCopyNov24(): Toggle
+
+    enum class Cohorts(override val cohortName: String) : CohortName {
+        CONTROL("control"),
+        PROTECTION("protection"),
```

Comment:
The cohort names are confusing. We have PIR but there's no copy related to PIR. All the copy is around VPN so I'd change the cohort names to avoid confusing when reviewing results.